src/gurt/telemetry.c

```diff
@@ -1093,14 +1093,20 @@ d_tm_set_counter(struct d_tm_node_t *metric, uint64_t value)
 		return;
 	}
 
+	if (unlikely(metric->dtn_protect))
```

Comment:
Just wondering if we should really optimize with `unlikely` for now. If we decided we needed to enable the mutexes more widely, we will be in a situation of needing to change these everywhere.

Reply:
That's the norm for now since nobody is using the mutex. I think that if the mutex is enabled for a few metrics, then we can afford the unlikely cost TBH.